### ISSUE TITLE
Error during initialization after update

### ISSUE THREAD

**Issue by miniBill** [lobanov/elm-taskport#13]
After updating to elm-taskport 2.0.0 I get this error during initialization:

```SyntaxError: Importing binding name 'default' cannot be resolved by star export entries.```

The stack trace is:
```
[N] link
[N] linkAndEvaluateModule
[N] (anonymouse function)
[N] promiseReactionJob
```

I'm running the code inside a webview managed by tauri-studio

**Comment by lobanov** [comment]
Thanks for reporting. I switched to WebPack bundler for the minified version in v2, which may have something to do with it. How are you including TaskPort JS file?

**Comment by miniBill** [comment]
```ts
import TaskPort from "elm-taskport";
```

In my `index.ts`. I'm using vite for development

**Comment by lobanov** [comment]
I suspect there may be a prolem with the exports. I will look into it, but in the meantime please can you try this:

```ts
import * as TaskPort from "elm-taskport";
```

**Comment by miniBill** [comment]
That works!

**Comment by lobanov** [comment]
Oh, that's good. In which case I'll just update the documentation. Thanks for checking.

**Comment by miniBill** [comment]
[feel free to close it after you update the docs]

**Comment by lobanov** [comment]
Released in 2.0.1
